client/src/components/web-extension/CreateWallet/useCreateUser.js

```diff
@@ -36,20 +38,21 @@ const useCreateUser = () => {
 		};
 	}, []);
 
-  /**
-   * Init new User account.
-   * Aim to return publicKey and User info, including hash info and User info
-   */
+	/**
+	 * Init new User account.
+	 * Aim to return publicKey and User info, including hash info and User info
+	 */
 	const onInitNewUserHandler = useCallback(
 		async (user) => {
-      // Get basic User info
+			// Get basic User info
 			const result = await onGetUserInfoHash(user);
 
-			// Create Wallet and get public key
+			// Create Wallet and get public key, public address
 			const wallet = await user.addWalletAccount(0, new WalletDescriptor('Account 1'));
 			const publicKey = await wallet.getPublicKey();
+			const publicAddress = await wallet.getPublicAddress();
```

Comment:
hi @duc-q @djhahe , why do we need to keep both publicKey and publicAddress?

Reply:
Updated code, I've used public address as public key for external UI

Reply:
but this is a code smell, this indicates we should clear up both on technical term and documentation because we're using one for another purpose

Reply:
Updated code, I've used public address as public key for external UI.

This is a code smell, this indicates we should clear up both on technical term and documentation because we're using one for another purpose.